### PR TITLE
Loading progress: Fixed missing content-length from server

### PR DESCRIPTION
**Pull request by carldebilly**

### CODE REVIEW

File: src/Uno.Wasm.Bootstrap/WasmScripts/uno-bootstrap.js [hunk 1]
```diff
@@ -223,6 +203,11 @@ var App = {
 
         asset = asset.replace("/managed/", "/" + config.uno_remote_managedpath + "/");
 
+        const assemblyName = asset.substring(asset.lastIndexOf('/') + 1);
+        if (config.assemblies_with_size.hasOwnProperty(assemblyName)) {
+            return this.fetchWithProgress(asset, (loaded, adding) => this.reportAssemblyLoading(adding));
+        }
+
         return fetch(asset, { credentials: 'same-origin' });
```
Comment (carldebilly) [inline src/Uno.Wasm.Bootstrap/WasmScripts/uno-bootstrap.js]: @jeromelaban Do you think it is required to keep that in the other `fetch`?

Comment (carldebilly) [reply src/Uno.Wasm.Bootstrap/WasmScripts/uno-bootstrap.js]: ... I mean the `'same-origin'`

Comment (jeromelaban) [reply src/Uno.Wasm.Bootstrap/WasmScripts/uno-bootstrap.js]: Isn't it the default ?

File: src/Uno.Wasm.Bootstrap/Uno.Wasm.Bootstrap.csproj [hunk 2]
```diff
@@ -73,6 +73,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<!-- This is intentionnal! -->
```
Comment (jeromelaban) [inline src/Uno.Wasm.Bootstrap/Uno.Wasm.Bootstrap.csproj]:
```suggestion
		<!-- This file is built as part of the UnoInstallSDKTask_v0 task -->
```

**Pull request merged**